tgui/packages/tgui/interfaces/Orbit.js

```diff
@@ -140,6 +140,26 @@ export const Orbit = (props, context) => {
                 onInput={(_, value) => setSearchText(value)}
                 onEnter={(_, value) => orbitMostRelevant(value)} />
             </Flex.Item>
+            <Flex.Item>
+              <Divider vertical />
+            </Flex.Item>
+            <Flex.Item>
+              <Button
+                inline
+                color="transparent"
+                tooltip="Toggle Auto-Observe"
+                tooltipPosition="bottom-left"
+                selected={auto_observe}
+                icon={auto_observe === 1 ? "toggle-on" : "toggle-off"}
+                onClick={() => act('toggle_observe')} />
```

Comment:
It doesn't matter, but I just want the file to stay consistent.

```suggestion
                onClick={() => act("toggle_observe")} />
```